dev/TreeView/TreeViewNode.cpp

```diff
@@ -193,27 +193,33 @@ void TreeViewNode::OnItemsSourceChanged(const winrt::IInspectable& sender, const
 
 void TreeViewNode::OnItemsAdded(int index, int count)
 {
-    for (int i = index + count - 1; i >= index; i--)
+    if (m_itemsDataSource.Count() != static_cast<int>(Children().Size()))
```

Comment:
What's with the unchanged count check? When will we already have the right number of children?

Reply:
Maybe clarify with a comment here.